oat-sa/extension-tao-itemqti-pci
# Feat/tr 5105/sanitizer in Math pci

Related to: https://oat-sa.atlassian.net/browse/TR-5105
Converter from https://github.com/oat-sa/tao-core-sdk-fe/pull/163

**How to test:**
create item with Math PCI, set correct answer `４２`
switch between Question and Response
save, see on Payload that number is sanitized to `42`
![image](https://user-images.githubusercontent.com/25976342/214310808-2f34c150-244b-4bd7-8563-25d1c727b8a8.png)
open correct Answer - is set to `42`
go to Preview, put answer `４２` 
Submit and see that send `42`
<img width="1267" alt="image" src="https://user-images.githubusercontent.com/25976342/214311370-afe77941-0cd7-43d6-96e9-c2413f000d3a.png">
